site/src/pages/CreateWorkspacePage/CreateWorkspacePage.tsx

```diff
@@ -127,6 +127,7 @@ const CreateWorkspacePage: FC = () => {
           defaultOwner={me}
           defaultBuildParameters={defaultBuildParameters}
           error={createWorkspaceMutation.error}
+          resetMutation={createWorkspaceMutation.reset}
```

Comment:
For me, resetMutation does not say too much about its intention. Maybe `resetError` could be a better name?